worker.py

```diff
@@ -492,6 +516,19 @@ def get_device_group_info(nebula_connection_object, device_group_to_get_info):
                         print("failed reporting state to kafka - exiting")
                         os._exit(2)
 
+            if redis_host is not None:
+                try:
```

Comment:
This needs to replicate the logic of  `if monotonic_id_increase is True or report_on_update_only is False:` that exists in the kafka reporter